.pre-commit-config.yaml

```diff
@@ -8,7 +8,18 @@ repos:
     -   id: end-of-file-fixer
     -   id: check-yaml
     -   id: check-added-large-files
--   repo: https://github.com/psf/black
-    rev: 24.2.0
+
+-   repo: local
```

Comment:
This instructs `pre-commit` to use the local versions of hooks. It guarantees that the version of `black` thats run on `git commit` matches whats in Poetry's lock file.